physical/raft/config_test.go

```diff
@@ -50,6 +58,7 @@ func TestRaft_ParseConfig(t *testing.T) {
 				cfg.RaftLogVerifierEnabled = true
 				cfg.RaftLogVerificationInterval = defaultRaftLogVerificationInterval
 			},
+			wantWarns: ceOnlyWarnings("raft_log_verification_interval is less than the minimum allowed"),
```

Comment:
is this warning not present in enterprise?

Reply:
Oh great catch. Will remove this wrapper.

Reply:
Fixed in [5ddc1ea](https://github.com/hashicorp/vault/pull/26052/commits/5ddc1ea4c56283170e27055256f9399edfc7fcfa)